scg-system/pom.xml

```diff
@@ -192,6 +210,22 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>io.telicent.public</groupId>
+      <artifactId>fuseki-yaml-config</artifactId>
+      <version>${dependency.yaml-config}</version>
+      <exclusions>
```

Comment:
Doesn't need addressing in this PR since it applies to the `fuseki-yaml-config` repository but we should probably go back and (re)move the concrete `log4j` dependency (possibly making it `test` scope as previously discussed) in favour of an SLF4J dependency (if it doesn't already exist) and make sure there's no non-test code reliant on log4j